### ISSUE TITLE
Show file details for uploaded documents

### ISSUE THREAD

**Issue by dottorer** [lale-help/lale-help#433]
We need to show all available file details to the document owner. Ideally we can show this in a nice form as designed in #338.

![image](https://cloud.githubusercontent.com/assets/14217285/20844786/ccd157b6-b8c1-11e6-9173-892dad5e38e2.png)

Today we only show:

![image](https://cloud.githubusercontent.com/assets/14217285/20844804/e702a054-b8c1-11e6-99a7-30b37cf07fc9.png)

We should show:
- "pretty file name" (editable)
- Boolean for visibility (editable)
- full document name (i.e. "24234_xcsdd_3242.xlsx")
- doc size in KB
- MIME Type if available
- uploaded by / uploaded on

**Comment by dottorer** [comment]
@NolanChan @diegoaad This could be a good small improvement based on a user feedback call I just had...

**Comment by NolanChan** [comment]
Hi @dottorer , can I assume this specific issue just concerns the edit page? And if it does, how would you like the extra information to be shown on the page? Is there a similar page that I can look to?

**Comment by NolanChan** [comment]
Were you thinking of something like this?

![screen shot 2016-12-06 at 11 30 54 pm](https://cloud.githubusercontent.com/assets/1487616/20958745/10449aba-bc0c-11e6-8abb-d48aa1cc638e.png)

**Comment by diegoaad** [comment]
@NolanChan when talking to @dottorer I understood that we'd still allow only the Name and the option "Visible to all helpers" available for edition. But on top of that we should show the File info including doc name, size, mime type and uploaded by/on. This could be a new widget on top of the page, like this one:

![](https://cloud.githubusercontent.com/assets/14217285/20844786/ccd157b6-b8c1-11e6-9173-892dad5e38e2.png)

**Comment by dottorer** [comment]
@NolanChan Yep, this is what I am dreaming of. We would then have a reusable representation of any document with its core data displayed. I will create some mockups asap to make my thoughts more clear.

**Comment by dottorer** [comment]
@NolanChan On the page 'Edit document', the solution you shown above would be a quick stop gap and can be helpful. I would request that for the immutable fields (anything but pretty name and visibility), we do not show the frame around the field but only the value. Maybe it is possible to condense the info in 2 cols or something neater without too much work.

Also, on this specific page it would be great to have a button on the very right bottom spot that says 'Delete'. Since we currently do not allow for update the stored file, we require people to actually delete the document and create a brand new one. This sucks as any bookmarked URL will be broken, so I am opening a separate issue in #442 for that one.

**Comment by NolanChan** [comment]
Hey @dottorer , I'm trying to make a simple code representation of your mock up. Could I get that default Lale picture?

**Comment by dottorer** [comment]
Hi @NolanChan : were you thinking about the tulip icon? I think we could use this for it...

![laleiconreversed60](https://cloud.githubusercontent.com/assets/14217285/21173259/22a1ae52-c1d7-11e6-9570-cd504acde6c3.png)

I can send you the SVG directly as well...

**Comment by NolanChan** [comment]
![screen shot 2017-01-08 at 4 58 08 pm](https://cloud.githubusercontent.com/assets/1487616/21755157/be4467aa-d5c3-11e6-84bf-4b7e7ad9e01d.png)

Hi @dottorer , here's a rough draft of my mock up so far. Could you give me some pointers on how it is so far? I haven't coded in any of the back-end yet to show any actual values.
